doc/user/snmptrap.rst

```diff
@@ -199,3 +200,18 @@ a siren, have your display flash, etc., be creative ;).
 
    # mail the notification
    echo "$MAIL" | mail -s "$SUBJECT" $EMAILADDR
+
+.. _traps-mib-selection:
+
+Traps Mib Selection in BGP
+--------------------------
+
+Both :rfc:`4273` and [Draft-IETF-idr-bgp4-mibv2-11]_ MIBs define traps for
+dealing with up/down events and state transition. The user has the
+possibility to select the MIB he wants to receive traps from:
+
+.. clicmd:: [no] bgp snmp traps <rfc4273|bgp4-mibv2>
```

Comment:
We don't use `[no]` in documentation, please drop it.

Reply:
taken into account